src/main/java/appeng/core/features/registries/IfaceTermRegistry.java

```diff
@@ -0,0 +1,36 @@
+package appeng.core.features.registries;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import appeng.api.features.IIfaceTermRegistry;
+import appeng.api.util.IIfaceTermViewable;
+import appeng.parts.misc.PartInterface;
+import appeng.parts.p2p.PartP2PInterface;
+import appeng.tile.misc.TileInterface;
+
+public class IfaceTermRegistry implements IIfaceTermRegistry {
```

Comment:
May be it would be better to not use shortcuts in name `InterfaceTerminalRegistry` it's easy to understand the code

Reply:
I have it commented in the external interface. The whole point of the refactor was to not have long names and iface is clear imo

Reply:
dbtble point imo

Reply:
You are using IDE completion anyway.
Trying to "not use long names" often lead to comical situations.

Reply:
What comical situation is happening here? It's not like a player will see this anyways. And even so, the opportunity to put "comical" names in otherwise boring code is cool. not like this a professional project

I am replacing `InterfaceTerminalSupportedClassProvider`. I've used the term `Iface` / `IfaceTerm` consistently in the new code with some documentation on how it's used, so still I don't see this as a strong reason to switch over

Reply:
Here is points to not use shortened names
 - I know noting about code but name. An if I try to search something i'll write like `InterfaceTerminal` -> nothing found
 - Consistent typos because `Iface` word not present in any dictionary as a valid word
 - Knowledge about why this thing named with this strange composition of letters persisted only in one head

Especially `setDispRep` and `setSelfRep` is unreadable at all. I need to investigate whole data flow inside this variable to realize what they really means -> `setMachineIcon` and `setCraftableIcon`.

Reply:
Seems like I had comments on the client side but not the packet. that will be fixed, but currently it is unused by the interface  terminal.

You may feel free to call out missing documentation, but I will not be changing the name